zap/src/main/java/org/parosproxy/paros/core/scanner/ScannerListener.java

```diff
@@ -41,4 +41,7 @@
 
     // ZAP: Added notifyNewMessage
     void notifyNewMessage(HttpMessage msg);
+
+    // Added to notify reason for filtering message from scanning.
```

Comment:
Either add to after the license header or change to JavaDoc (inline comments are not useful, the ones above are for legacy reasons).

Reply:
sure.